### ISSUE TITLE
Installation Fails

### ISSUE THREAD

**Issue by RobBW** [league1991/CodeAtlasVsix#7]
On MacOS High Sierra, VSCode Version 1.31.0 (2019) installation fails with message: "extension/package.json not found inside zip."

**Comment by ciessielski** [comment]
same here

**Comment by RobBW** [comment]
Reading of the website again: https://github.com/league1991/CodeAtlasVsix suggests that CodeAtlas was developed for VS Studio not the more recent VS Code and that this is most likely the problem. Its a pity!

**Comment by ciessielski** [comment]
@league1991 any suggestions?

**Comment by janek** [comment]
@league1991 would be awesome to be able to use this in VSCode, would porting require much work?

**Comment by league1991** [comment]
@janek Yes, much work needs to be done because VSCode's extension API is totally different. I'm afraid I don't have such a plan in the near future.

**Comment by league1991** [comment]
Sorry guys, I won't support VSCode in the near future, because the API is totally different.

**Comment by janek** [comment]
gotcha @league1991, thank you for answering!

**Comment by RobBW** [comment]
Thanks for your response. 
Very understandable reason.
Pity though, you have good looking extension there.
Regards
Rob Wilkinson

**Comment by Kochise** [comment]
And vs2019 ?

**Comment by league1991** [comment]
@RobBW Thank you :-)

@Kochise Code Graph for Visual Studio 2019 is in https://marketplace.visualstudio.com/items?itemName=YaobinOuyang.CodeAtlas
